### PR TITLE
Add Madokami source

### PR DESCRIPTION
**Pull request by bobobo1618**

### CODE REVIEW

File: src/en/madokami/build.gradle [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    appName = 'Tachiyomi: Madokami'
+    pkgNameSuffix = 'en.madokami'
+    extClass = '.Madokami'
+    extVersionCode = 1
+    extVersionSuffix = 1
+    libVersion = '1.4'
```
Comment (AmineI) [inline src/en/madokami/build.gradle]: Here, the libVersion being set to 1.4 (??) while the last one is 1.2 is making the Travis build fail as you can see in Travis's log. I believe it's the only issue preventing the build.

Comment (arkon) [reply src/en/madokami/build.gradle]: It's supposed to go with https://github.com/inorichi/tachiyomi-extensions-lib/pull/4

Comment (AmineI) [reply src/en/madokami/build.gradle]: Oh right, didn't notice that pull request on the lib repo. Indeed, it was intended not to pass without that PR being accepted. Ignore my comment then, I'll see if I can edit/remove it, just wanted to help :) .

**Comment by as280093** [comment]
If the PR get fixed here are the res so merge while fixing

[res.zip](https://github.com/inorichi/tachiyomi-extensions/files/2937622/res.zip)
ic_launcher

**Pull request closed without merging**